docs/content/users/quickstart.md

```diff
@@ -323,19 +323,21 @@ ddev launch
 
 ## Shopware
 
-You can set up a Shopware 6 environment many ways, we recommend the following technique:
+You can set up a Shopware 6 environment many ways, we recommend the following technique. DDEV creates a .env.local file for you by default; if you already have one DDEV leaves it alone.
 
 ```bash
-git clone --branch=6.4 https://github.com/shopware/production my-shopware6
-cd my-shopware6
-ddev config --project-type=shopware6 --docroot=public
-ddev start
-ddev composer install --no-scripts
+mkdir my-shopware6 && cd my-shopware6
+ddev config --project-type=shopware6 --docroot=public --create-docroot
+ddev get ddev/ddev-elasticsearch
+ddev composer create shopware/production:^v6.5
+# If it asks `Do you want to include Docker configuration from recipes?` answer `x`, as we're setting it up.
 # During system:setup you may have to enter the Database user (db), Database password (db)
 # Database host (db) and Database name (db).
-ddev exec bin/console system:setup --database-url=mysql://db:db@db:3306/db --app-url='{DDEV_PRIMARY_URL}'
-ddev exec bin/console system:install --create-database --basic-setup
+rm .env # composer create creates a .env file, but we want to start with .env.local
+ddev exec console system:setup --database-url=mysql://db:db@db:3306/db --app-url='${DDEV_PRIMARY_URL}' --mailer-url='smtp://127.0.0.1:1025?encryption=&auth_mode='
+ddev exec console system:install --create-database --basic-setup
 ddev launch /admin
+# The default user/password are admin/shopware
```

Comment:
This may be excessively subjective, but I’d make this read more fluidly and wrap those literal values in backticks. Or at the very least, drop “the”:

```suggestion
# Default username and password are `admin` and `shopware`
```

Reply:
Or maybe “Default credentials” if that’s not too vague.